pkg/controller/common/nodenetwork.go

```diff
@@ -121,3 +120,36 @@ func CreateAllNodeNetworkIfNotExist(networkType networkv1.NetworkType, cn *netwo
 
 	return nil
 }
+
+// if cluster network is open, set the default NIC for all node networks.
+// if cluster network is close, clear the NIC specification of all node networks.
+func SetNICForAllNodeNetworks(enable bool, nic string, cache ctlnetworkv1.NodeNetworkCache, client ctlnetworkv1.NodeNetworkClient) error {
```

Comment:
define a common struct instead of passing the cache and client directly.

Reply:
We passed the cache and client directly for all functions in this common file. Reserve this style to avoid affecting other functions in this PR and I will submit another PR to optimize it after the 0.3 release. What do you think?

Reply:
sure